src/test/java/uk/gov/hmcts/reform/judicialapi/elinks/controller/ElinksControllerTest.java

```diff
@@ -71,7 +71,7 @@ void test_load_location_success() {
 
         ResponseEntity<ElinkBaseLocationWrapperResponse> actual = eLinksController.loadLocation();
         assertThat(actual).isNotNull();
-        assertThat(actual.getStatusCodeValue()).isEqualTo(HttpStatus.OK.value());
+        assertThat(actual.getStatusCode().value()).isEqualTo(HttpStatus.OK.value());
```

Comment:
minor: would be more readable to compare enum values to each other (same for other assertions below)